src/ui/dom/ListView.ts

```diff
@@ -80,7 +80,8 @@ class IteratorList<T> extends OnDemandList {
 
 		var widget:SingleNodeWidget = new Ctor({
 			app: this._app,
-			model: model
+			model: model,
+			parent: this
```

Comment:
Currently parent isn’t expected to be set until the widget is actually inserted by insertRow